manifests/params.pp

```diff
@@ -1,6 +1,8 @@
 # Class prometheus::params
 # Include default parameters for prometheus class
-class prometheus::params {
+class prometheus::params (
+  $extract_command = undef,
```

Comment:
this should not be a parameter of the params class

Reply:
Happy to change it. Can you please advise how it should be structured/refactored?

Reply:
this should be a normal variable and not a parameter.